app/domain/table_displays/event/participations/question_column.rb

```diff
@@ -44,7 +44,7 @@ def sort_by(attr)
     protected
 
     def allowed?(object, _attr, _original_object, _original_attr)
-      ability.can?(:update, object.event) || ability.can?(:show_full, object.person)
+      ability.can?(:index_full_participations, object.event) || ability.can?(:show_full, object.person)
```

Comment:
@carlobeltrame Weisst du noch, weshalb hier `can?(:show_full, object.person)` geprüft wird (ebenfalls in ShowDetails/FullOrEventLeaderColumn)? Dahin gelangt ein User ja nur, wenn er `:index_full_participations` nicht hat. Warum soll er dann für einzelne TN diese Informationen doch sehen? Oder geht es nur darum, die eigenen Daten zu sehen?  Ich sehe den Use Case dafür nicht, selbst wenn der User via Gruppen/Rolle Zugriff auf diese Daten hätte (wobei dies bei den Event Questions sowieso nicht der Fall ist, nur bei Geschlecht, Geburtsdatum, usw).

Als Quelle habe ich nur dies gefunden: https://help.puzzle.ch/#ticket/zoom/5905